libr/include/r_bin_dwarf.h

```diff
@@ -110,23 +111,24 @@ extern "C" {
 #define DW_TAG_partial_unit             0x3c  /* DWARF3 */
 #define DW_TAG_imported_unit            0x3d  /* DWARF3 */
         /*  Do not use DW_TAG_mutable_type */
-#define DW_TAG_mutable_type 0x3e /* Withdrawn from DWARF3 by DWARF3f. */
+#define DW_TAG_mutable_type 			0x3e /* Withdrawn from DWARF3 by DWARF3f. */
```

Comment:
also here

Reply:
Yeah you're right. It looked okay in my editor, just noticed it's completely broken on github